proto/google/events/firebase/analytics/v1/events.proto

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package google.events.firebase.analytics.v1;
+
+import "google/events/firebase/analytics/v1/data.proto";
+import "google/events/cloudevent.proto";
+
+option csharp_namespace = "Google.Events.Protobuf.Firebase.Analytics.V1";
+
+// The CloudEvent raised when a Firebase Analytics log is written.
+message AnalyticsLogWritten {
```

Comment:
At one point I think we want to have "Event" suffix for messages in `events.proto`?

Reply:
Yes - will fix.